editors/code/src/main.ts

```diff
@@ -182,6 +182,8 @@ async function getServer(config: Config, state: PersistentState): Promise<string
         }
         return explicitPath;
     };
+    const releaseTag = config.extensionReleaseTag;
+    if (releaseTag === null) return "rust-analyzer";
```

Comment:
I'd suggest not making this speciall `null` case and put `nightly` release tag in `package.json` by default. This way while developing the frontend we are not required to have a local build of the server which is what I like.